omlmd/helpers.py

```diff
@@ -7,6 +7,7 @@
 from dataclasses import fields
 from pathlib import Path
 
+from omlmd.constants import FILENAME_METADATA_JSON, FILENAME_METADATA_YAML, MIME_APPLICATION_CONFIG, MIME_APPLICATION_MLMODEL
```

Comment:
suggestion: maybe `from omlmd import constants as const` to reduce this section? or just split the import by lines

```suggestion
from omlmd.constants import (
    FILENAME_METADATA_JSON,
    FILENAME_METADATA_YAML,
    MIME_APPLICATION_CONFIG,
    MIME_APPLICATION_MLMODEL,
)
```

to tidy a little?